src/main/java/org/opentripplanner/graph_builder/module/StreetLinkerModule.java

```diff
@@ -57,7 +57,10 @@ public void buildGraph(
     if (graph.hasStreets) {
       linkTransitStops(graph);
       linkTransitEntrances(graph);
-      linkBikeParks(graph);
+      if (!graph.hasLinkedBikeParks) {
+        linkBikeParks(graph);
+        graph.hasLinkedBikeParks = true;
+      }
```

Comment:
Should the `if (!graph.hasLinkedBikeParks)` be moved inside the `linkBikeParks` method, or du we force this in another place?

It probably deserves a comment (on the method if moved inside or inline if not)

Reply:
Done